src/handler.ts

```diff
@@ -46,16 +50,14 @@ export async function handlePullRequest (context: Context): Promise<void> {
     }
   }
 
-  if (config.addAssignees && reviewers.length > 0) {
-    try {
-      const assignees: string[] = config.assignees ?
-        chooseUsers(owner, config.assignees, config.numberOfAssignees || config.numberOfReviewers)
-        :
-        reviewers
+  const assignees: string[] = chooseUsers(
```

Comment:
I'm sorry but it must be the previous condition to maintain compatibility.

There was no assignees field until the PR #37 was submitted, and the same value was added into assignee based on the user selected from the reviewers list.

In the near future I want to review this.

```yml
# Set to true to add reviewers to pull requests
addReviewers: true

# Set to true to add assignees to pull requests
addAssignees: true

# A list of reviewers to be added to pull requests (GitHub user name)
reviewers:
  - reviewerA
  - reviewerB
  - reviewerC
```

Reply:
I'll check it out in the evening - thank you for the input.

Reply:
I'm ok with maintaining the previous behavior. It felt inconsistent, why sometimes you would end up with assignees with the pr creator, and sometimes without.

On the other hand, I can see the consistency in "if you won't setup the 'assignees', you'll end up with the behavior of the reviewers".

I'll adjust accordingly.

Reply:
> I'm ok with maintaining the previous behavior.

Thanks for your work.

>  It felt inconsistent, why sometimes you would end up with assignees with the pr creator, and sometimes without.

Surely, there is inconsistent. It's ok to change to the same behavior as the reviewers

> On the other hand, I can see the consistency in "if you won't setup the 'assignees', you'll end up with the behavior of the reviewers".

I agree with your opinion. Until now I missed the timing to fix it.

Could you adjust as is possible for you to do?

Reply:
I've just realized that with the config you've mentioned, you'll end up with the same result with both implementations (it doesn't touch the "prCreator" issue). Please see my latest comment on the matter.